File: DotNetTest/MessageTest.cs
/*=============================================================
(c) 2010 by The Build Doctor Limited
All rights reserved.
================================================================*/

namespace BuildDoctor{
  using System;
  using NUnit.Framework;
  [TestFixture]
   public class MessageTest{
      [Test] 
      public void ShouldHaveMysteronVoices() 
        {
         
         Assert.AreEqual ("This is the voice of the Mysterons", new Message().Text());
        
        }
      
      
   }
   
   
}

File: JavaTest/src/test/java/com/builddoctor/MessageTest.java
package com.builddoctor;

import com.builddoctor.Message;
import org.junit.Test;
import junit.framework.Assert;

public class MessageTest {
	@Test
	public void shouldReturnMysteronRelatedTextWhenIAskIt() {
		Message message = new Message();
		Assert.assertEquals("This is the voice of the Cybermen", message.messageText());
		
	}
}